src/game/settings/Settings.hpp

```diff
@@ -0,0 +1,148 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman
+** File description:
+** Profile
+*/
+
+#ifndef GAME_PROFILE_HPP_
+#define GAME_PROFILE_HPP_
+
+#include <filesystem>
+#include <stdexcept>
+#include <string>
+#include "raylib/core/Vector2.hpp"
+#include <string_view>
+
+namespace game
+{
+    namespace settings
+    {
+        /// Settings common to the game
+        class Settings {
+          public:
+            /// Exception for invalid values in settings.
+            class InvalidSettingsValue : public std::logic_error {
```

Comment:
`InvalidSettingsValues` should inherit `std::runtime_error`, since it can be triggered by the users